packages/icons/README.md

```diff
@@ -83,7 +83,20 @@ const SuccessMessage = ({ description }) => (
 );
 ```
 
-It is not possible to change the color of an external SVG using the `css` color property. Instead, you can use the [CSS `filter` hack](https://blog.union.io/code/2017/08/10/img-svg-fill/) to colorize the icon.
+#### Color your icon
```

Comment:
What's the motivation behind these changes? 

When importing SVGs as files (as opposed to React components which is documented above) it's not possible to style them with CSS.

Reply:
Right, the color change is properly documented above!
I will revert this change, thanks for pointing out!